test_docopt.py

```diff
@@ -431,7 +428,7 @@ def test_language_errors():
     with raises(DocoptLanguageError):
         docopt('no usage with colon here')
     with raises(DocoptLanguageError):
-        docopt('usage: here \n\n and again usage: here')
+        docopt('first usage: here \n\nsecond usage: here')
```

Comment:
Because of a space before 'and' the section looked like:
```
usage: here 

 and again usage: here
```
So, the second line was indented and therefore considered to be a part of the first section.